mne_bids/mne_bids.py

```diff
@@ -753,11 +753,20 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     # KIT data requires the marker file to be copied over too
     if 'mrk' in raw._init_kwargs:
         hpi = raw._init_kwargs['mrk']
-        _, marker_ext = _parse_ext(hpi)
-        marker_fname = make_bids_basename(
-            subject=subject_id, session=session_id, task=task, run=run,
-            acquisition=acquisition, suffix='markers%s' % marker_ext,
-            prefix=os.path.join(data_path, bids_raw_folder))
-        sh.copyfile(hpi, marker_fname)
+        acq_map = dict()
+        if isinstance(hpi, list):
+            # order hpi list by acquisition time
+            hpi.sort(key=_get_mrk_meas_date)
+            _, marker_ext = _parse_ext(hpi[0])
+            acq_map = dict(zip(['pre', 'post'], hpi))
+        else:
+            _, marker_ext = _parse_ext(hpi)
+            acq_map[None] = hpi
+        for key, value in acq_map.items():
+            marker_fname = make_bids_basename(
+                subject=subject_id, session=session_id, task=task, run=run,
+                acquisition=key, suffix='markers%s' % marker_ext,
+                prefix=os.path.join(data_path, bids_raw_folder))
```

Comment:
this is not defined before?

Reply:
This is the same as how we had the code, just moved into a loop